src/stylesheets/components/_modal.scss

```diff
@@ -0,0 +1,175 @@
+.usa-js-no-click {
+  pointer-events: none;
+  user-select: none;
+
+  .usa-modal,
+  .usa-modal * {
+    pointer-events: auto;
+    user-select: text;
+  }
+}
+
+// Prevents propagation issue in Safari an opener
+// has a child element, and when clicked, prevents
+// intended action
+[data-open-modal] * {
+  pointer-events: none;
+}
+
+.usa-modal-wrapper {
+  text-align: center;
+  transition: opacity $project-easing;
+
+  // Using visibility: hidden instead of display: none because
+  // of iOS VoiceOver issue:
+  // https://developer.paciellogroup.com/blog/2018/06/the-current-state-of-modal-dialog-accessibility/
+  &.is-hidden {
+    visibility: hidden;
+    opacity: opacity(0);
+    position: fixed;
+  }
+
+  &.is-visible {
+    visibility: visible;
+    opacity: opacity(100);
+    position: fixed;
+    z-index: z-index($theme-modal-z-index);
+  }
+}
+
+.usa-modal-overlay {
+  background: rgba(0, 0, 0, 0.5);
```

Comment:
```suggest
  background: color("black-transparent-50");
```

Reply:
Had no idea that token existed!

Reply:
I wonder if it'd be beneficial to provide settings for `usa-modal-overlay` and `usa-modal` to allow users to re-theme it for dark mode.